deoxys/src/lib.rs

```diff
@@ -321,3 +324,115 @@ where
     B: DeoxysBcType,
 {
 }
+
+#[cfg(all(feature = "alloc", test))]
```

Comment:
If you put these tests in the toplevel `tests/` directory they automatically get std/alloc

Reply:
yeah but I can't put them in tests I don't have access to the subkeys there.

Reply:
Update this is now the last commit in https://github.com/RustCrypto/AEADs/pull/665